packages/app-project/pages/_document.js

```diff
@@ -43,7 +43,7 @@ export default class MyDocument extends Document {
 
   render () {
     return (
-      <Html lang="en">
+      <Html lang={this.props.locale}>
```

Comment:
```suggestion
      <Html>
```

Next.js i18n-routing [automatically sets](https://nextjs.org/docs/advanced-features/i18n-routing#search-engine-optimization) the html `lang` tag, so I think we can remove any direct manipulation of it.